Pod/Classes/Logging/Log.swift

```diff
@@ -74,37 +113,100 @@ open class Log {
+                logger.info("\(logMessage):\n\(objectString, privacy: .private)")
+            case .warn:
+                logger.warning("\(logMessage):\n\(objectString, privacy: .private)")
+            case .error:
+                logger.error("\(logMessage):\n\(objectString, privacy: .private)")
+            case .off:
+                break
+            }
+        }
+        else {
+            let nameString = name.count > 0 ? "[\(name)]]" : ""
+            print(nameString + logString + "\n")
         }
+        self.handler?(level, logString)
+        Log.globalHandler?(self, level, logString)
     }
 
-    // MARK: Log Methods
+    // MARK: Public
 
-    public static func error<T>(_ object: @autoclosure () -> T, _ fileName: String = #file, _ functionName: String = #function, _ line: Int = #line) {
+    public func error<T>(_ object: @autoclosure () -> T, _ fileName: String = #file, _ functionName: String = #function, _ line: Int = #line) {
         log(object(), level:.error, fileName, functionName, line)
     }
 
-    public static func warn<T>(_ object: @autoclosure () -> T, _ fileName: String = #file, _ functionName: String = #function, _ line: Int = #line) {
+    public func warn<T>(_ object: @autoclosure () -> T, _ fileName: String = #file, _ functionName: String = #function, _ line: Int = #line) {
         log(object(), level:.warn, fileName, functionName, line)
     }
 
-    public static func info<T>(_ object: @autoclosure () -> T, _ fileName: String = #file, _ functionName: String = #function, _ line: Int = #line) {
+    public func info<T>(_ object: @autoclosure () -> T, _ fileName: String = #file, _ functionName: String = #function, _ line: Int = #line) {
         log(object(), level:.info, fileName, functionName, line)
     }
 
-    public static func debug<T>(_ object: @autoclosure () -> T, _ fileName: String = #file, _ functionName: String = #function, _ line: Int = #line) {
+    public func debug<T>(_ object: @autoclosure () -> T, _ fileName: String = #file, _ functionName: String = #function, _ line: Int = #line) {
         log(object(), level:.debug, fileName, functionName, line)
     }
 
-    public static func verbose<T>(_ object: @autoclosure () -> T, _ fileName: String = #file, _ functionName: String = #function, _ line: Int = #line) {
+    public func verbose<T>(_ object: @autoclosure () -> T, _ fileName: String = #file, _ functionName: String = #function, _ line: Int = #line) {
         log(object(), level:.verbose, fileName, functionName, line)
     }
 }
+
+// MARK: Static Helper Methods
+extension Log {
+
+    public static var logLevel: Level {
+        get { instance.logLevel }
+        set { instance.logLevel = newValue }
+    }
+
+    public static var useEmoji: Bool {
+        get { instance.useEmoji }
+        set { instance.useEmoji = newValue }
+    }
+
+    public static var handler: InstanceLogHandler? {
+        get { instance.handler }
+        set { instance.handler = newValue }
+    }
```

Comment:
Oh! I see the compatibility shim is down here. Might be good to mark some of these as deprecated

Reply:
do we want them deprecated?  For apps where a single Log instance is sufficient, I can still see some value keeping these shorthand versions around

Reply:
Yeah perhaps not deprecated, but specifically the change related to the renaming of `handler` and `globalHandler` maybe. Also this one in particular is a bit confusing because it's an `InstanceLogHandler` but applied globally

Reply:
it's applied to the shared instance, but not necessarily globally.  I agree the name is a little confusing though 🤔 
Mainly, this is here for backwards compatibility.d